src/app/app.tsx

```diff
@@ -18,7 +18,7 @@ import { Devtools } from '@app/features/devtool/devtools';
 import { AppRoutes } from '@app/routes/app-routes';
 import { persistor, store } from '@app/store';
 
-const reactQueryDevToolsEnabled = process.env.REACT_QUERY_DEVTOOLS_ENABLED;
+const reactQueryDevToolsEnabled = process.env.REACT_QUERY_DEVTOOLS_ENABLED === 'true';
```

Comment:
This should be removed?

Reply:
No, it was an error before. The values come in as strings and `Boolean("false") === true`

Reply:
O, got it, thx.